src/dialects/oracle/schema/trigger.js

```diff
@@ -20,6 +20,7 @@ const trigger = {
       `    FROM all_constraints cons, all_cons_columns cols` +
```

Comment:
Another suggestion is to query from `user_constraints` and `user_cons_columns` instead of the "all" variants. This will limit the scope to the current user/schema only.